src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -211,11 +212,27 @@ public static function createConnection(string $dsn, array $options = [])
                     if (!isset($params['redis_sentinel'])) {
                         break;
                     }
-                    $extra = [];
-                    if (\defined('Redis::OPT_NULL_MULTIBULK_AS_NULL') && isset($params['auth'])) {
-                        $extra = [$params['auth']];
+
+                    if (version_compare(phpversion('redis'), '6.0.0') >= 0) {
```

Comment:
We generally use the signature of version_compare using 3 arguments (which passes an operator and returns a bool) as it makes the code easier to read (not having to remember which case returns 1 and which case returns -1)